src/LoginComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class LoginComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.loginUser();
+  }
+
+  loginUser = () => {
+    axios.post('/api/login', {
+      email: this.state.email,
+      password: this.state.password
+    })
+    .then((response) => {
```

Comment:
In case of an error, the error is logged to the console but the user is not informed about the failure of the login operation. Consider adding error handling and user feedback.